sdks/java/core/src/main/java/org/apache/beam/sdk/schemas/transforms/SchemaTransformProvider.java

```diff
@@ -58,10 +59,14 @@ default String description() {
   SchemaTransform from(Row configuration);
 
   /** Returns the input collection names of this transform. */
-  List<String> inputCollectionNames();
+  default List<String> inputCollectionNames() {
```

Comment:
intellij thinks these methods are unused - is intellij wrong? or could they be removed?

Reply:
So if they override these methods but we never call them, does it matter?

Reply:
Ahh these methods are eventually called by the expansion service to create the discover response. Reference here: 
https://github.com/apache/beam/blob/fedca3ce18ea21424bd2381c75231e7399a8a6c2/sdks/java/expansion-service/src/main/java/org/apache/beam/sdk/expansion/service/ExpansionService.java#L733-L734